### PR TITLE
Add instructions to setup common observing environment in nublado.

### PR DESCRIPTION
**Pull request by edennihy**
This has an accompanying PR in auxtel_obsrun_env_setups to modify the user setup script at https://github.com/lsst-sitcom/auxtel_obsruns_env_setups/pull/28

### CODE REVIEW

File: Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+.. code-block:: shell-session
+
+  $ cd auxtel_obsruns_env_setups
+  $ bash bash/lsst/sitcom/user_setups_mod.sh
+    Moving .user_setups file to /home/edennihy/notebooks/user_setups-2023-05-03T04:44:40.bak 
+    Creating a new ~/notebooks/.user_setups file 
+
+    New .user_setup file created 
+
+The above message contains the location of the backup version of you current .user_setups file that was overwritten ``/home/edennihy/notebooks/user_setups-2023-05-03T04:44:40.bak``.
```
Comment (tribeiro) [inline Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst:98]: Suggested rephrase:

```
The above message shows where the users previous ``.user_setups`` file can be found, e.g.; ``/home/edennihy/notebooks/user_setups-2023-05-03T04:44:40.bak``.
```

File: Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  $ cd auxtel_obsruns_env_setups
+  $ bash bash/lsst/sitcom/user_setups_mod.sh
+    Moving .user_setups file to /home/edennihy/notebooks/user_setups-2023-05-03T04:44:40.bak 
+    Creating a new ~/notebooks/.user_setups file 
+
+    New .user_setup file created 
+
+The above message contains the location of the backup version of you current .user_setups file that was overwritten ``/home/edennihy/notebooks/user_setups-2023-05-03T04:44:40.bak``. 
+
+Your new .user_setup file will only contain the minimum commands necessary to setup your notebook environment with the Common Observing Environment and you DDS Daemon.
```
Comment (tribeiro) [inline Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst:100]: Suggested edit:

```
Note that the new ``.user_setup`` file only contains the minimum commands necessary to setup the Common Observing Environment.
```

File: Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  git https://github.com/lsst-sitcom/auxtel_obsruns_env_setups.git 
+
+Enter the following sequence of commands to run the automatic setup script. 
+
+.. warning::
+    
+    This step will create a new .user_setups file and overwrite and backup your current .user_setups file. 
+    The backup copy will be tagged with a timestamp version in the same directory.  
+    If you have any custom environment setup commands in your current .user_setups file, 
+    you will need to manually add them to the new .user_setups file that will be created.
```
Comment (tribeiro) [inline Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst:87]: Merge this line with the previous as it a continuation.

**Review by tribeiro — APPROVED** [review]
Looks good! I left a few comments for you to consider.

File: Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+.. _Setup-Common-Observing-Environment-in-Nublado-Post-Conditions:
+
+Post-Condition
+==============
+
+
+- User notebooks will source packages directly from the common environment installed at 
+
+.. prompt:: bash
+    /opt/obs_env/auto_base_packages/
```
[the remark this answers is not in the document]
Comment (edennihy) [reply Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst:37]: Removed reference to path here.

File: Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+In the terminal, check to see if you already have either of the following directories installed in your home space:
+
+.. code-block:: shell-session
+
+  auxtel_obsruns_env_setups
+  auto-op-env-packages
+
+If they already exist, remove both directories. 
+
+In the terminal enter the following sequence of commands to git clone the common observing environment setup package which contains the automatic setup script;
```
[the remark this answers is not in the document]
Comment (edennihy) [reply Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst:74]: Change adopted.

File: Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst (hunk 2 above)
Comment (edennihy) [reply Observing-Interface-Setup/setup-common-observing-environment-in-nubaldo.rst:100]: Suggestion adopted.

**Pull request merged**